saleor/order/emails.py

```diff
@@ -43,6 +45,36 @@ def collect_data_for_email(order_pk, template):
     }
 
 
+def collect_staff_data_for_email(order_pk, template):
+    """Collect the required data for sending emails.
+
+    Args:
+        order_pk (int): order primary key
+        template (str): email template path
+
+    """
+    order = Order.objects.get(pk=order_pk)
+    staff_notifications = StaffNotification.objects.filter(active=True)
```

Comment:
Shouldn't we check that users from `staff_notifications` are still `active==True`?

Reply:
true, will fix

Reply:
true, will fix